src/LinuxTracing/UprobesUnwindingVisitor.cpp

```diff
@@ -477,16 +479,39 @@ void UprobesUnwindingVisitor::Visit(uint64_t event_timestamp, const MmapPerfEven
   }
 
   // Don't try to send a ModuleUpdateEvent for non-executable maps.
-  if (!event_data.executable) {
-    return;
-  }
-  // Don't try to send a ModuleUpdateEvent for anonymous maps.
-  if (event_data.filename.empty() || event_data.filename[0] == '[') {
-    return;
+  if (!event_data.executable) return;
```

Comment:
Hehe unfortunately we don't have an internal guideline on this and we are not consistent, so it will always be down to preference.
As you can see from the existing code that was also from me and had braces, I'm also inconsistent with myself.
I'm adding the braces, it's on me I removed them.
:DDD